src/main/java/frc/robot/subsystems/arm/ArmIOReal.java

```diff
@@ -161,5 +172,9 @@ public void update() {
 
     armMotorConnectedAlert.set(!armConnected);
     armEncoderConnectedAlert.set(!armEncoder.isConnected());
+
+    if (m_emergencyMode == true) {
```

Comment:
Following Rob's suggestion, move the `setVoltage(0)` method directly to `setEmergencyMode` instead. Instead of waiting for periodic updates, it will drop the voltage down to 0 for safety reasons as well